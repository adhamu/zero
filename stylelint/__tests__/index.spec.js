const stylelintConfig = require('..')

describe('Stylelint config', () => {
  it('returns the correct config', () => {
    expect(stylelintConfig).toMatchSnapshot()
  })
})
