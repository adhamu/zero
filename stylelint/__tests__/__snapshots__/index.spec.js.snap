// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stylelint config returns the correct config 1`] = `
{
  "extends": [
    "@stylistic/stylelint-config",
    "stylelint-config-standard-scss",
    "stylelint-config-idiomatic-order",
  ],
  "overrides": [
    {
      "files": [
        "**/*.css",
      ],
      "rules": {
        "at-rule-no-unknown": true,
      },
    },
    {
      "files": [
        "**/*.scss",
      ],
      "rules": {
        "at-rule-no-unknown": [
          true,
          {
            "ignoreAtRules": [
              "use",
              "forward",
              "include",
              "mixin",
            ],
          },
        ],
        "max-nesting-depth": [
          2,
          {
            "ignoreAtRules": [
              "each",
              "media",
              "supports",
              "include",
            ],
          },
        ],
      },
    },
  ],
  "plugins": [
    "stylelint-scss",
  ],
  "rules": {
    "@stylistic/function-parentheses-space-inside": null,
    "@stylistic/string-quotes": [
      "single",
      {
        "avoidEscape": true,
      },
    ],
    "declaration-property-unit-disallowed-list": {
      "font-size": [
        "px",
      ],
      "line-height": [
        "px",
      ],
    },
    "length-zero-no-unit": true,
    "order/order": [
      [
        "custom-properties",
        "dollar-variables",
        {
          "name": "extend",
          "type": "at-rule",
        },
        "declarations",
        {
          "hasBlock": false,
          "name": "include",
          "type": "at-rule",
        },
        "rules",
      ],
    ],
    "selector-no-qualifying-type": [
      true,
      {
        "ignore": [
          "attribute",
          "class",
          "id",
        ],
      },
    ],
  },
}
`;
